.github/workflows/test-helpers/get-issue-validation-log

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+function download-and-extract {
+  issue_number="$1"
+  echo "Determining workflow run ID for issue #${issue_number} ..."
+
+  # It seems that for issues, run_number equals issue_number - 1 (i.e., it's
+  # zero-indexed whereas issue number is one-indexed). I'm not seeing this
+  # actually documented anywhere so it could be that this breaks at some point.
+  # If/when that happens, we should notice due to the hard check at the end of
+  # this function, and at that point I guess we'll need to fall back on a
+  # clunkier strategy of downloading and inspecting the log for each run until
+  # we find the issue number.
+
+  rm -rf log.zip log
+  run_id=$(
+    gh api "repos/:owner/:repo/actions/workflows/validate-new-issue.yml/runs" \
+      | jq ".workflow_runs[] 
+            | select((.event==\"issues\") and (.run_number==$(( issue_number - 1 )))) 
```

Comment:
Picking up from https://github.com/getsentry/.github/pull/21/files#r563509632 ...

> I'd rather stick to the clunky-but-documented way instead of using this unproven link.

We do have a check at the end of this function to validate this assumption about `run_number`. I think in the interest of time we should proceed with the code as written. If it works, great! If we see it start breaking then we can implement the clunkier approach (downloading, extracting, and inspecting logs for each run until we find the right one).